pages/asks.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useMemo } from "react";
+import styles from "@/styles/pages/asks.module.scss";
+import Ask from "@/components/Ask";
+import content from "@/server/content";
+import CreateAskForm from "@/components/CreateAskForm";
+import Modal from "@/components/Modal";
+
+export default function Asks() {
+	const contentArr = Object.entries(content);
+	const [showOnlyLikedAsks, setShowOnlyLikedAsks] = useState(false);
+	const [isModalOpen, setIsModalOpen] = useState(false);
+
+	const handleAskClick = () => {
+		setIsModalOpen((prev) => !prev);
+	};
+
+	const toggleLike = () => {
+		setShowOnlyLikedAsks((prev) => !prev);
+	};
+
+	const askElements = useMemo(() => {
+		return contentArr.reduce((acc, [id, askContent]) => {
+			if (!showOnlyLikedAsks || (showOnlyLikedAsks && askContent.like)) {
+				acc.push(
+					<Ask
+						key={id}
+						id={id}
+						ask={askContent.ask}
+						isLiked={askContent.like}
+						isAnonymous={askContent.anonymous}
+						userId={askContent.userId}
+					/>
+				);
+			}
+
+			return acc;
+		}, []);
+	}, [showOnlyLikedAsks]);
+
+	return (
+		<main className={styles.main}>
```

Comment:
In my opinion, would be more descriptive to identify all the pages' main element as `<page-name>-page` as its the root element/style of the page.

Reply:
I believe it's better the way it is right now, it clarify that you style the main content of the page, and not part of it.